core/src/main/scala/chisel3/RawModule.scala

```diff
@@ -296,6 +296,19 @@ abstract class RawModule extends BaseModule {
   protected[chisel3] override def moduleBuilt(): Unit = _afterModuleBuilt.foreach(_())
 
   private[chisel3] def initializeInParent(): Unit = {}
+
+  final var _knownLayers: Option[Seq[layer.Layer]] = None
```

Comment:
Side note and probably moot since ideally we can get rid of this, but if we do need to keep it, it needs to be private.